.github/workflows/build-test.yml

```diff
@@ -75,6 +75,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: pre-flight-check
     steps:
+      - run: sudo apt-mark hold grub-efi-amd64-signed
```

Comment:
@dkroenke Could you please elaborate a bit under the section `Notes for Reviewer` on what this change is about?

Reply:
@mossmaurice See https://github.com/ros-tooling/setup-ros/issues/542